src/components/Simulation/Filters.js

```diff
@@ -0,0 +1,71 @@
+import React from "react";
+import PropTypes from "prop-types";
+import { withTranslation } from "react-i18next";
+import MultiSelectDropdown from "../Shared/MultiSelectDropdown";
+import { formattedName } from "../../utils/textUtils";
+
+export const SimulationFilters = props => {
+  // Renders the Filters of a simulation, a Simulation exists out of
+  // multiple parts, this component will allow us to filter
+  // those. Each part is identified by the combination:
+  //
+  //       - orgUnit
+  //       - package
+  //       - period
+  //
+  // It accepts callbacks for the `change` event, so the actual
+  // hiding/showing is handled in a higher component.
+
+  const {
+    allPeriods,
+    periods,
+    periodsChanged,
+    allOrgUnits,
+    orgUnits,
+    orgUnitsChanged,
+    allPackages,
+    packages,
+    packagesChanged,
+    t,
+  } = props;
```

Comment:
In this case, since only used once, could make sense to not déstructure the props.

Reply:
I kind of like how you can read all the props in a predictable place.

Reply:
Yep, but you technically have them all in the prototypes as well

Reply:
Non blocking for me though, it’s mostly to have less lines